packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -223,7 +224,29 @@ export class DocumentDeltaConnection
 	}
 
 	private checkNotClosed() {
-		assert(!this.disposed, 0x20c /* "connection disposed" */);
+		// Increase the stack trace limit temporarily, so as to debug better in case it occurs.
+		// We are seeing this in telemetry and we are unable to figure out why it is happening, so this should help.
+		const originalStackTraceLimit = (Error as any).stackTraceLimit;
+		try {
+			(Error as any).stackTraceLimit = 50;
+			assert(!this.disposed, 0x20c /* "connection disposed" */);
```

Comment:
I'd remove try/catch and assert, and simply use sendErrorEvent - it will collect stack, and that's where stackTraceLimit  would be useful.
try/catch has stronger guarantees re clean-up, but I'd say it's fine if sendErrorEvent throws and all stacks are longer after that point - we should simply remove this code once we know for sure what happens here.